src/cell_draw.cpp

```diff
@@ -563,12 +574,26 @@ void draw_character_sprite_in_water(
     // Upper body
     pos(x + 24, y + 16);
     gmode(2);
-    gcopy_c(texture_id, frame, direction * 48, 32, 28, 24, 24);
+    gcopy_c(
+        texture_id,
+        frame,
+        direction * 48,
+        32,
+        28,
+        pcc_size(24, 32),
+        pcc_size(24, 28));
 
     // Lower body
-    pos(x + 24, y + 36);
+    pos(x + 24, y + 16 + pcc_size(16, 20));
```

Comment:
If size is shrinked, it would be 32 instead of 36, the old value.

Reply:
Thanks. I'll fix.